src/rngs/mod.rs

```diff
@@ -0,0 +1,184 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// https://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Various RNGs and wrappers.
+//!
+//! This module provides the three standard RNGs exposed by Rand (besides
+//! specific PRNG algorithms in the `prng` module). Those are [`ThreadRng`],
+//! [`StdRng`], and [`SmallRng`].
+//!
+//! To get a seed for those PRNGs at runtime, [`EntropyRng`], [`OsRng`] and
+//! [`JitterRng`] are sources of external randomness. Those are mostly
+//! implementation details for Rand, prefer to use the [`FromEntropy`] trait
```

Comment:
This implies there are no direct uses of `OsRng` — we've previously recommended using it directly for important keys.

Reply:
I'll add a note.

Reply:
Using `OsRng` is also essentially what crypto libraries like ring do.